baselines/dom.generated.d.ts

```diff
@@ -2677,6 +2678,7 @@ declare var Document: {
 }
 
 interface DocumentFragment extends Node, NodeSelector {
+    parentNode: parentNode;
```

Comment:
this should be added to the extends clause and not a property. the spec says `DocumentFragment` implements `ParentNode`, and not that it has a `parentNode` property, all `Node`s have a parentNode alaredy. unless i am missing something